src/strconv.cxx

```diff
@@ -178,7 +178,7 @@ bool valid_infinity_string(const char str[]) noexcept
 template<typename T> class dumb_stringstream : public std::stringstream
 {
 public:
-  dumb_stringstream() : std::stringstream{}
+  dumb_stringstream() : std::stringstream()
```

Comment:
Today {} sometimes might be a little different thing than ().
In constructor preferred way is (), as I understand.